### ISSUE TITLE
Reorganize discussion/conclusion

### ISSUE THREAD

**Issue by timothydmorton** [RuthAngus/GProtation#17]
The discussion/conclusion currently goes like this:

### Discussion
* We underestimate uncertainties
  * mostly due to multi-modality
  * might be due to kernel choice
* QP kernel is effective though simplistic; one might consider cos*SE
  * Would negative covariances be OK?
  * Formal model comparison floated; considered unfeasible
* Sometimes stars don't show a rotation signal; other times we may find a "false positive"
  * Several suggestions for identifying false positives (giant detection, strange hyperparameters)
* We are thinking about several other things
  * Model selection with different kernel functions
  * Design and implement physically motivated kernel function
  * Attempt to detect differential rotation
  * Build a noise model for kepler data. 

### Conclusion
* We implement and test our method; it does better than the others
* Our method produces uncertainties, so that's great
  * But we still don't really trust them
  * Maybe because the MCMC did not converge?
* Though GP model is good, it is still only an effective model, not a physically accurate one
  * Only a quarter of the uncertainties are "accurate"
* Main aim of this work is probabilistic rotation period inference
  * Could use for hierarchical inference
  * Though it still has issues, and is only "effective" and we don't really trust the uncertainties
  * But, it's probabilistic and more accurate, so we still think it's the best.

I think this could use some reorganization.  I don't have a specific suggestion yet, but perhaps we can jot outline ideas in this thread?  I think we also need to figure out what our message is regarding the uncertainties (and, maybe more importantly, figure out why we really think they are being underestimated).  Perhaps we could move discussion of the uncertainties to Section 3?  If we did that then the "Discussion" would basically be only a discussion about a different possible kernel, and the ideas for future work (which might work better in the Conclusion?)... Just brainstorming here...

**Comment by RuthAngus** [comment]
Yes - moving the discussion of uncertainties to section three is a good idea - I like that! I think it would be nice to find out the underlying cause and to try to modify the method to do better, but I'm also desperate to get this paper submitted so I'm reluctant to do any more analysis...
